examples/feast/simple_feature_store/feature_repo/run.py

```diff
@@ -0,0 +1,43 @@
+from datetime import datetime
+
+import store_definitions
+import store_operations
+from demo_inputs import ONLINE_ENTITY_ROWS  # noqa: F401
+from demo_inputs import ONLINE_FEATURES  # noqa: F401
+from demo_inputs import STREAM_EVENT_DF  # noqa: F401
+from demo_inputs import HISTORICAL_ENTITY_DF, HISTORICAL_FEATURES
+
+from hamilton import base, driver
+
+
+def main():
+    dr = driver.Driver(
+        dict(feast_repository_path=".", feast_config={}),
+        store_operations,
+        store_definitions,
+        adapter=base.SimplePythonGraphAdapter(base.DictResult()),
+    )
+
+    final_vars = [
+        "apply"
+        # "driver_activity_v1_fs",
```

Comment:
I want people to be able to switch to the other sets of values by uncommenting them. I can making "uncommentable" from outside `main()` or define a small click CLI. In the notebook, it will be different cells

Reply:
sure, make a comment here then?